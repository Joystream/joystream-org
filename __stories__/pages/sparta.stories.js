import React from 'react';
import { storiesOf } from '@storybook/react';

import SpartaPage from '../../src/pages/sparta';

storiesOf('Pages|Sparta', module).add('default', () => <SpartaPage />);
