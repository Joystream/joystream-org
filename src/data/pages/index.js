import React from 'react';
import { ReactComponent as validatorsImage } from '../../assets/svg/active-validators.svg';
import { ReactComponent as storageImage } from '../../assets/svg/platform-content-files.svg';
import { ReactComponent as memberImage } from '../../assets/svg/council-member.svg';
import { ReactComponent as screenerImage } from '../../assets/svg/membership-screener.svg';
import { ReactComponent as membershipCuratorImage } from '../../assets/svg/membership-curator.svg';
import { ReactComponent as contentCuratorImage } from '../../assets/svg/content-curator.svg';
import { ReactComponent as contentCreatorImage } from '../../assets/svg/content-creator.svg';
import { ReactComponent as bandwidthProviderImage } from '../../assets/svg/bandwidth-provider.svg';
import { ReactComponent as discoveryProviderImage } from '../../assets/svg/discovery-provider.svg';
import { ReactComponent as liveStreamingProviderImage } from '../../assets/svg/live-streaming-provider.svg';
import { ReactComponent as builderImage } from '../../assets/svg/builder.svg';
import { ReactComponent as communicationModeratorImage } from '../../assets/svg/communication-moderator.svg';

export const sharedData = {
  defaultEmail: 'hello@jsgenesis.com',
  social: {
    telegramLink: 'https://t.me/joinchat/CNyeUxHD9H56m3e_44hXIA',
    redditLink: 'https://www.reddit.com/r/JoyStream/',
    twitterLink: 'https://twitter.com/JoyStreamApp',
  },
  links: {
    blog: 'https://blog.joystream.org/',
    whitepaper: 'https://github.com/Joystream/whitepaper/blob/master/paper.pdf',
    repository: 'https://github.com/Joystream/joystream',
    github: 'https://github.com/Joystream',
    openings: 'https://www.jsgenesis.com/#openings',
  },
  rolesDescription: (
    <>
      Explore available roles and pick the one that suits you best. Influence the platform's development and earn Monero
      in the process.
    </>
  ),
};

export const roles = {
  active: [
    {
      image: validatorsImage,
      title: 'Validator',
      to: '/roles#validator',
      key: 'validatorsCount',
      hasLabel: false,
    },
    {
      image: memberImage,
      title: 'Council Member',
      to: '/roles#council-member',
      key: 'councilMembersCount',
      hasLabel: false,
    },
    {
      image: storageImage,
      title: 'Storage Provider',
      to: '/roles#storage-provider',
      key: 'storageProviders',
      hasLabel: true,
    },
  ],
  future: [
    {
      image: screenerImage,
      title: 'Membership Screener',
      to: '/roles#membership-screener',
    },
    {
      image: membershipCuratorImage,
      title: 'Membership Curator',
      to: '/roles',
    },
    {
      image: contentCuratorImage,
      title: 'Content Curator',
      to: '/roles#content-curator',
    },
    {
      image: contentCreatorImage,
      title: 'Content Creator',
      to: '/roles#content-creator',
    },
    {
      image: bandwidthProviderImage,
      title: 'Bandwidth Provider',
      to: '/roles#bandwidth-provider',
    },
    {
      image: discoveryProviderImage,
      title: 'Discovery Provider',
      to: '/roles#discovery-provider',
    },
    {
      image: liveStreamingProviderImage,
      title: 'Live Streaming Provider',
      to: '/roles#live-streaming-provider',
    },
    { image: builderImage, title: 'Builder', to: '/roles#builder' },
    {
      image: communicationModeratorImage,
      title: 'Communication Moderator',
      to: '/roles#communication-moderator',
    },
  ],
};
