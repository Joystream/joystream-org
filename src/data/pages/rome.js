const goalsData = [
  {
    title: 'Build a dynamic, flexible and user friendly content directory',
    text: `The Rome testnet is built around the introduction of a new content 
      directory, that can be added to, modified and improved without requiring 
      runtime upgrades. All key results and goals are derived from this.`,
    state: 'inprogress',
  },
  {
    title: 'Populate the content directory with a variety of content',
    text: `Rome introduces a Content Directory system that is both built to last, 
      yet still meant to be continuously improved on. We will pay for Content Creators 
      to upload a variety of content and we hope to learn a lot from the results.`,
    state: 'inprogress',
  },
  {
    title: 'Introduce the new Content Curator role',
    text: `With a new Content Directory, and incentives to upload content, 
      we need monitoring. The Content Curators will require staking, and 
      will be paid in both testnet tokens and Monero.`,
    state: 'inprogress',
  },
  {
    title: 'Add Content Creator profiles',
    text: `Without Content Creators, there won't be any content. Granted, 
      adding another level on top of the membership requirement may seem 
      like a hurdle, but it will allow users to upload with multiple 
      profiles.`,
    state: 'inprogress',
  },
];

const launchDate = '2019/12/16 11:00';

export { goalsData, launchDate };
