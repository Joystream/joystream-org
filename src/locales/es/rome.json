{
  "rome": {
    "siteMetadata": {
      "description": "Explora la testnet de Rome"
    },
    "hero": {
      "title": "Red de Rome",
      "text": "Explore available roles and pick the one that suits you best. Influence the platform's development and earn real money in the process.",
      "chipText": "What is this?"
    },
    "heroCard": {
      "title": "REEMPLAZADO POR CONSTANTINOPLE EN"
    },
    "modal": {
      "title": "Rome",
      "text": "<0>The Roman empire left many landmarks.</0> Their architecture and engineering skills was unparalleled during their might. The concept of aqueducts was not devised by the Romans, but their beauty and extent is."
    },
    "criticalDocuments": {
      "title": "Documentos críticos",
      "fullSpecifications": {
        "title": "Full Specifications",
        "text": "No specifications were published for Rome."
      },
      "releasePlan": {
        "title": "Plan de lanzamiento",
        "text": "Lea el plan de lanzamiento tal como se hizo durante la etapa de planificación y aprenda más sobre cómo evolucionó el desarrollo."
      }
    },
    "testnetGoals": {
      "title": "Objetivos de la testnet",
      "subtitle": "The goals below are a simplified representation of the Key Results listed in our Release <0>OKR</0>",
      "goals": {
        "contentDirectory": {
          "title": "Build a dynamic, flexible and user friendly content directory",
          "text": "The Rome testnet is built around the introduction of a new content directory, that can be added to, modified and improved without requiring runtime upgrades. All key results and goals are derived from this."
        },
        "content": {
          "title": "Populate the content directory with a variety of content",
          "text": "Rome introduces a Content Directory system that is both built to last, yet still meant to be continuously improved on. We will pay for Content Creators to upload a variety of content and we hope to learn a lot from the results."
        },
        "contentCurator": {
          "title": "Introduce the new Content Curator role",
          "text": "With a new Content Directory, and incentives to upload content, we need monitoring. The Content Curators will require staking, and will be paid in both testnet tokens and Monero."
        },
        "contentCreator": {
          "title": "Add Content Creator profiles",
          "text": "Without Content Creators, there won't be any content. Granted, adding another level on top of the membership requirement may seem like a hurdle, but it will allow users to upload with multiple profiles."
        }
      }
    },
    "roles": {
      "title": "Funciones incentivadas para la red de Rome"
    },
    "map": {
      "title": "Rome",
      "text": "<0>Rome was the capital of the great Roman Empire during its peak.</0> As with previous testnet names, the Roman Empire and Rome was another important step in the rise of democracy, the rule of law, and modern governance structures.<1/><1/><2><0/> Explore current testnet</2>"
    }
  }
}
