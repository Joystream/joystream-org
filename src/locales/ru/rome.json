{
  "rome": {
    "siteMetadata": {
      "description": "Изучаем тестнет Rome"
    },
    "hero": {
      "title": "Сеть Rome",
      "text": "Изучайте доступные роли и выбирайте наиболее подходящие вам. Влияйте на развитие платформы, зарабатывая при этом реальные деньги.",
      "chipText": "Что это?"
    },
    "heroCard": {
      "title": "ЗАМЕНЕНА НА CONSTANTINOPLE"
    },
    "modal": {
      "title": "Rome",
      "text": "<0>Римская империя оставила богатое наследие.</0> Во время ее расцвета архитектурные и инженерные навыки римлян не имели себе равных. Хотя идея акведуков не была изобретена римлянами, их красота и протяженность поражают."
    },
    "criticalDocuments": {
      "title": "Важная документация",
      "fullSpecifications": {
        "title": "Полные спецификации",
        "text": "Спецификации тестнета Rome не были опубликованы."
      },
      "releasePlan": {
        "title": "План релиза",
        "text": "Прочтите план релиза в том виде, в котором он был составлен на этапе планирования, и сравните с тем, как прошла разработка."
      }
    },
    "testnetGoals": {
      "title": "Цели тестнета",
      "subtitle": "Указанные ниже цели дают общее представление о ключевых результатах, перечисленных в нашем релизе <0>OKR</0>.",
      "goals": {
        "contentDirectory": {
          "title": "Создание динамичного, гибкого и удобного каталога содержимого",
          "text": "Краеугольным камнем тестнета Rome является введение нового каталога контента, который можно расширять, изменять и улучшать без необходимости обновления среды выполнения. Им и обусловлены все основные цели и результаты."
        },
        "content": {
          "title": "Заполнение каталога различным контентом",
          "text": "Мы выпускаем в Rome новую систему каталога контента, которая, с одной стороны, рассчитана на длительный срок службы, а с другой - на постоянные улучшения. Мы будем оплачивать загрузку различного содержимого создателям контента, надеясь, что результаты нас многому научат."
        },
        "contentCurator": {
          "title": "Введение новой роли куратора контента",
          "text": "С введением нового каталога контента и поощрений за публикации нам необходим мониторинг. Чтобы стать куратором контента, необходимо будет выполнить процедуру стейкинга. Кураторы будут получать оплату как в токенах тестнета, так и в Monero."
        },
        "contentCreator": {
          "title": "Добавление профилей создателей контента",
          "text": "Без создателей контента не было бы никакого контента. Разумеется, добавление дополнительной роли поверх членства может создать определенный барьер, но это позволит пользователям загружать контент, используя разные профили."
        }
      }
    },
    "roles": {
      "title": "Поощряемые роли в сети Rome"
    },
    "map": {
      "title": "Рим",
      "text": "<0>Рим был столицей великой Римской империи в период ее расцвета.</0> Подобно названиям предыдущих тестнетов, Римская империя и Рим были еще одним важным шагом в развитии демократии, верховенства закона и современных структур управления.<1/><1/><2><0/> Информация о тестнете</2>"
    }
  }
}
