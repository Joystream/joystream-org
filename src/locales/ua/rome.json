{
  "rome": {
    "siteMetadata": {
      "description": "Explore the Rome testnet"
    },
    "hero": {
      "title": "Rome Network",
      "text": "Explore available roles and pick the one that suits you best. Influence the platform's development and earn real money in the process.",
      "chipText": "What is this?"
    },
    "heroCard": {
      "title": "REPLACED BY CONSTANTINOPLE ON"
    },
    "modal": {
      "title": "Rome",
      "text": "<0>The Roman empire left many landmarks.</0> Their architecture and engineering skills was unparalleled during their might. The concept of aqueducts was not devised by the Romans, but their beauty and extent is."
    },
    "criticalDocuments": {
      "title": "Critical Documents",
      "fullSpecifications": {
        "title": "Full Specifications",
        "text": "No specifications were published for Rome."
      },
      "releasePlan": {
        "title": "Release Plan",
        "text": "Read the release plan as it was made during the planning stage, and learn more about how the development evolved."
      }
    },
    "testnetGoals": {
      "title": "Testnet Goals",
      "subtitle": "The goals below are a simplified representation of the Key Results listed in our Release <0>OKR</0>",
      "goals": {
        "contentDirectory": {
          "title": "Build a dynamic, flexible and user friendly content directory",
          "text": "The Rome testnet is built around the introduction of a new content directory, that can be added to, modified and improved without requiring runtime upgrades. All key results and goals are derived from this."
        },
        "content": {
          "title": "Populate the content directory with a variety of content",
          "text": "Rome introduces a Content Directory system that is both built to last, yet still meant to be continuously improved on. We will pay for Content Creators to upload a variety of content and we hope to learn a lot from the results."
        },
        "contentCurator": {
          "title": "Introduce the new Content Curator role",
          "text": "With a new Content Directory, and incentives to upload content, we need monitoring. The Content Curators will require staking, and will be paid in both testnet tokens and Monero."
        },
        "contentCreator": {
          "title": "Add Content Creator profiles",
          "text": "Without Content Creators, there won't be any content. Granted, adding another level on top of the membership requirement may seem like a hurdle, but it will allow users to upload with multiple profiles."
        }
      }
    },
    "roles": {
      "title": "Incentivized Roles for the Rome Network"
    },
    "map": {
      "title": "Rome",
      "text": "<0>Rome was the capital of the great Roman Empire during its peak.</0> As with previous testnet names, the Roman Empire and Rome was another important step in the rise of democracy, the rule of law, and modern governance structures.<1/><1/><2><0/> Explore current testnet</2>"
    }
  }
}
