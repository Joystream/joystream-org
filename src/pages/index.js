import React from 'react';
import { pagePropTypes } from '../propTypes';

import getApiPath from '../utils/getApiPath';
import mapStatusDataToAnalytics from '../utils/mapStatusDataToAnalytics';
import mapStatusDataToRoles from '../utils/mapStatusDataToRoles';

import withApi from '../components/_enhancers/withApi';

import BaseLayout from '../components/_layouts/Base';
import Link from '../components/Link';
import Subheader from '../components/Subheader';
import TestnetItem from '../components/TestnetItem';
import Analytics from '../components/Analytics';
import Button from '../components/Button';
import TitleWrapper from '../components/TitleWrapper';
import RoleList from '../components/RoleList';
import ColumnsLayout from '../components/ColumnsLayout';
import LayoutWrapper from '../components/LayoutWrapper';
import Hero from '../components/Hero';
import SiteMetadata from '../components/SiteMetadata';

import AcropolisImage from '../assets/svg/acropolis-main.svg';
import heroImage from '../assets/svg/hero-builder.svg';
import RomeImage from '../assets/svg/rome-main.svg';
import { ReactComponent as TickImage } from '../assets/svg/tick.svg';
import { ReactComponent as ClockImage } from '../assets/svg/clock.svg';

import { roles } from '../data/pages';
import { launchDate as romeNetworkLaunchDate } from '../data/pages/rome';

import './style.scss';

const activeTestnet = {
  name: 'Acropolis',
  incentivesLink: 'https://blog.joystream.org/acropolis-incentives/',
};

const IndexPage = ({ content }) => (
  <BaseLayout>
    <SiteMetadata
      title="Joystream: A user governed video platform"
      description="Earn monero for participating on our testnets!"
    />
    <Hero image={heroImage} title="A user-governed video platform" animationStartValue={0}>
      <p className="IndexPage__hero-paragraph">
        Earn Monero by participating in the current {activeTestnet.name} testnet
      </p>
      <div className="IndexPage__hero-group">
        <Button noWrap large secondary className="IndexPage__hero-button" href={activeTestnet.incentivesLink}>
          Earn Monero
        </Button>
        <Button
          noWrap
          large
          secondary
          reversed
          className="IndexPage__hero-button"
          href="https://testnet.joystream.org/"
        >
          Launch UI
        </Button>
      </div>
    </Hero>

    <LayoutWrapper>
      <TestnetItem
        title="Acropolis Testnet"
        image={AcropolisImage}
        children={
          <>
            Acropolis is our fourth testnet, with much improved{' '}
            <Link to="/roles#storage-provider">storage provider</Link> software and an on-chain forum.
          </>
        }
        button={{
          label: 'Explore Acropolis',
          to: '/acropolis',
        }}
      />
    </LayoutWrapper>

    <Analytics title="Testnet Metrics" large content={mapStatusDataToAnalytics(content)}>
      <Button secondary href={activeTestnet.incentivesLink}>
        Participate and Earn Monero
      </Button>
    </Analytics>

    <LayoutWrapper>
      <TestnetItem
        title="Rome Testnet"
        image={RomeImage}
        children="Rome is our fifth testnet, and we are introducing new roles for you to try out."
        date={romeNetworkLaunchDate}
        button={{
          label: 'Explore Rome',
          to: '/rome',
        }}
      />
    </LayoutWrapper>

    <LayoutWrapper dark>
      <TitleWrapper title="Become a user, run the platform">
        <Subheader
          title="Active roles on the current testnet"
          content="learn more, join a role and subscribe for more"
          icon={TickImage}
        />
        <ColumnsLayout>
          <RoleList roles={roles.active} content={mapStatusDataToRoles(content)} />
        </ColumnsLayout>
        <Subheader
          title="Roles coming in future testnets"
          content="choose a role, learn more and subscribe to get updated"
          icon={ClockImage}
        />
        <ColumnsLayout columnsCount={3}>
          <RoleList roles={roles.future} content={mapStatusDataToRoles(content)} />
        </ColumnsLayout>

        <Button secondary to="/roles" className="IndexPage__roles-button">
          Learn more about the roles
        </Button>
      </TitleWrapper>
    </LayoutWrapper>
  </BaseLayout>
);

IndexPage.propTypes = pagePropTypes;

export { IndexPage };
export default withApi(IndexPage, getApiPath('STATUS'));
