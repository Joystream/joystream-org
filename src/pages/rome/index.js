import React, { useState } from 'react';
import { pagePropTypes } from '../../propTypes';

import getApiPath from '../../utils/getApiPath';
import mapStatusDataToRoles from '../../utils/mapStatusDataToRoles';

import withApi from '../../components/_enhancers/withApi';

import BaseLayout from '../../components/_layouts/Base';
import HeroCard from '../../components/HeroCard';
import TitleWrapper from '../../components/TitleWrapper';
import RoleList from '../../components/RoleList';
import ColumnsLayout from '../../components/ColumnsLayout';
import LayoutWrapper from '../../components/LayoutWrapper';
import Hero from '../../components/Hero';
import TestnetModal from '../../components/TestnetModal';
import Chip from '../../components/Chip';
import Pane from '../../components/Pane';
import GoalList from '../../components/GoalList';
import Link from '../../components/Link';
import MapInfo from '../../components/MapInfo';
import SiteMetadata from '../../components/SiteMetadata';

import romeImage from '../../assets/svg/rome.svg';
import { ReactComponent as SpecImg } from '../../assets/svg/specifications.svg';
import { ReactComponent as ReleaseImg } from '../../assets/svg/release-doc.svg';
import { ReactComponent as PersonIcon } from '../../assets/svg/person.svg';
import RomeBuildingImg from '../../assets/svg/rome-building.svg';

import { roles, sharedData } from '../../data/pages';
import { goalsData, launchDate } from '../../data/pages/rome';

import './style.scss';

const RomePage = ({ content }) => {
  const [isModalOpen, setModalClosed] = useState(false);

  return (
    <BaseLayout>
      <SiteMetadata title="Joystream: The video platform DAO" description="Explore the Rome Testnet" />

      <Hero
        image={romeImage}
        title="Rome Network"
        indent
        chip={<Chip onClick={() => setModalClosed(true)}>What is this?</Chip>}
        animationStartValue={0}
      >
        <p className="RomePage__hero-paragraph">{sharedData.rolesDescription}</p>
        <HeroCard date={launchDate} />

        <TestnetModal
          title="Rome"
          image={RomeBuildingImg}
          closeModal={() => setModalClosed(false)}
          isOpen={isModalOpen}
        >
          <p>
            <strong>The Roman empire left many landmarks.</strong> Their architecture and engineering skills was
            unparalleled during their might. The concept of aqueducts was not devised by the Romans, but their beauty
            and extent is.
          </p>
        </TestnetModal>
      </Hero>

      <LayoutWrapper>
        <TitleWrapper title="Critical Documents">
          <ColumnsLayout>
            <Pane image={SpecImg} disabled title="Full Specifications">
              Specs to be released
            </Pane>
            <Pane
              image={ReleaseImg}
              href="https://github.com/Joystream/joystream/tree/master/testnets/rome"
              title="Release Plan"
              target="_blank"
            >
              Read the release plan as it was made during the planning stage, and learn more about how the development
              evolved.
            </Pane>
          </ColumnsLayout>
        </TitleWrapper>

        <TitleWrapper
          title="Testnet Goals"
          subtitle={
            <>
              The goals below are a simplified representation of the Key Results listed in our Release{' '}
              <Link href="https://github.com/Joystream/joystream/tree/master/testnets/rome#release-okrs">OKR</Link>
            </>
          }
        >
          <GoalList data={goalsData} />
        </TitleWrapper>
      </LayoutWrapper>

      <LayoutWrapper dark>
        <TitleWrapper title="Incentivized Roles for the Rome Network">
          <ColumnsLayout>
            <RoleList roles={roles.active} content={mapStatusDataToRoles(content)} />
          </ColumnsLayout>
        </TitleWrapper>
      </LayoutWrapper>

      <MapInfo title="Rome" location="rome">
        <p>
          <strong>Rome was the capital of the great Roman Empire during its peak.</strong> As with previous testnet
          names, the Roman Empire and Rome was another important step in the rise of democracy, the rule of law, and
          modern governance structures.
          <br />
          <br />
          <Link href="https://testnet.joystream.org/">
            <PersonIcon /> Explore current testnet
          </Link>
        </p>
      </MapInfo>
    </BaseLayout>
  );
};

RomePage.propTypes = pagePropTypes;

export { RomePage };
export default withApi(RomePage, getApiPath('STATUS'));
