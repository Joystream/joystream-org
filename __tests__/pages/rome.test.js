import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import { RomePage } from '../../src/pages/rome';
import testContent from '../../__mocks__/data/testContent';

describe('RomePage page', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<RomePage content={testContent} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
