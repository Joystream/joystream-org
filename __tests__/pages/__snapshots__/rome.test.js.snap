// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RomePage page renders correctly 1`] = `
<BaseLayout>
  <SiteMetadata
    description="Explore the upcoming Rome Testnet"
    image="test-file-stub"
    lang="en"
    meta={Array []}
    title="Joystream: A user governed video platform"
  />
  <Hero
    animationEnd={500}
    animationEndValue={-40}
    animationStartValue={0}
    chip={
      <Chip
        icon="icon-component-mock"
        onClick={[Function]}
      >
        What is this?
      </Chip>
    }
    image="icon-mock"
    indent={true}
    noOverflow={false}
    theme="black"
    title="Rome Network"
  >
    <p
      className="RomePage__hero-paragraph"
    >
      Explore available roles and pick the one that suits you best. Influence the platform's development and earn Monero in the process.
    </p>
    <HeroCard
      content={null}
      counterTitle=""
      date="2019/12/16 11:00"
      error={false}
      info={false}
    />
    <TestnetModal
      className=""
      closeModal={[Function]}
      image="icon-mock"
      isOpen={false}
      title="Rome"
    >
      <p>
        <strong>
          The Roman empire left many landmarks.
        </strong>
         Their architecture and engineering skills was unparalleled during their might. The concept of aqueducts was not devised by the Romans, but their beauty and extent is.
      </p>
    </TestnetModal>
  </Hero>
  <LayoutWrapper
    className=""
    dark={false}
    gradient={false}
  >
    <TitleWrapper
      className=""
      subtitle=""
      title="Critical Documents"
    >
      <ColumnsLayout
        className=""
        columnsCount={2}
        largeSpacing={false}
      >
        <Pane
          className=""
          disabled={true}
          image="icon-component-mock"
          title="Full Specifications"
        >
          Specs to be released
        </Pane>
        <Pane
          className=""
          disabled={false}
          href="https://github.com/Joystream/joystream/tree/master/testnets/rome"
          image="icon-component-mock"
          target="_blank"
          title="Release Plan"
        >
          Read the release plan as it was made during the planning stage, and learn more about how the development evolved.
        </Pane>
      </ColumnsLayout>
    </TitleWrapper>
    <TitleWrapper
      className=""
      subtitle={
        <React.Fragment>
          The goals below are a simplified representation of the Key Results listed in our Release
           
          <Link
            className=""
            highlighted={false}
            href="https://github.com/Joystream/joystream/tree/master/testnets/rome#release-okrs"
          >
            OKR
          </Link>
        </React.Fragment>
      }
      title="Testnet Goals"
    >
      <GoalList
        data={
          Array [
            Object {
              "state": "inprogress",
              "text": "The Rome testnet is built around the introduction of a new content 
      directory, that can be added to, modified and improved without requiring 
      runtime upgrades. All key results and goals are derived from this.",
              "title": "Build a dynamic, flexible and user friendly content directory",
            },
            Object {
              "state": "inprogress",
              "text": "Rome introduces a Content Directory system that is both built to last, 
      yet still meant to be continuously improved on. We will pay for Content Creators 
      to upload a variety of content and we hope to learn a lot from the results.",
              "title": "Populate the content directory with a variety of content",
            },
            Object {
              "state": "inprogress",
              "text": "With a new Content Directory, and incentives to upload content, 
      we need monitoring. The Content Curators will require staking, and 
      will be paid in both testnet tokens and Monero.",
              "title": "Introduce the new Content Curator role",
            },
            Object {
              "state": "inprogress",
              "text": "Without Content Creators, there won't be any content. Granted, 
      adding another level on top of the membership requirement may seem 
      like a hurdle, but it will allow users to upload with multiple 
      profiles.",
              "title": "Add Content Creator profiles",
            },
          ]
        }
      />
    </TitleWrapper>
  </LayoutWrapper>
  <LayoutWrapper
    className=""
    dark={true}
    gradient={false}
  >
    <TitleWrapper
      className=""
      subtitle=""
      title="Incentivized Roles for the Rome Network"
    >
      <ColumnsLayout
        className=""
        columnsCount={2}
        largeSpacing={false}
      >
        <RoleList
          content={
            Object {
              "councilMembersCount": 12,
              "storageProviders": 10,
              "validatorsCount": 20,
            }
          }
          roles={
            Array [
              Object {
                "hasLabel": false,
                "image": "icon-component-mock",
                "key": "validatorsCount",
                "title": "Validator",
                "to": "/roles#validator",
              },
              Object {
                "hasLabel": false,
                "image": "icon-component-mock",
                "key": "councilMembersCount",
                "title": "Council Member",
                "to": "/roles#council-member",
              },
              Object {
                "hasLabel": true,
                "image": "icon-component-mock",
                "key": "storageProviders",
                "title": "Storage Provider",
                "to": "/roles#storage-provider",
              },
            ]
          }
        />
      </ColumnsLayout>
    </TitleWrapper>
  </LayoutWrapper>
  <MapInfo
    className=""
    location="rome"
    title="Rome"
  >
    <p>
      <strong>
        Rome was the capital of the great Roman Empire during its peak.
      </strong>
       As with previous testnet names, the Roman Empire and Rome was another important step in the rise of democracy, the rule of law, and modern governance structures.
      <br />
      <br />
      <Link
        className=""
        highlighted={false}
        to="/acropolis"
      >
        <icon-component-mock />
         Explore current testnet
      </Link>
    </p>
  </MapInfo>
</BaseLayout>
`;
